code/game/machinery/embedded_controller/access_controller.dm

```diff
@@ -159,22 +159,21 @@
 		closeDoor(A)
 
 /obj/machinery/doorButtons/airlock_controller/proc/closeDoor(obj/machinery/door/airlock/A)
-	set waitfor = FALSE

```

Comment:
Was this change involved in fixing the bug?

Reply:
Yes. The proc is used to evaluate an if statement, having it async messed with that. The reason it was there originally was because airlocks start waiting if something is in the way and they can't close properly. By making them unsafe upon closing we override that behaviour, and there is no need to have this async anymore.